cirq-core/cirq/qis/clifford_tableau.py

```diff
@@ -652,3 +652,6 @@ def measure(
         self, axes: Sequence[int], seed: 'cirq.RANDOM_STATE_OR_SEED_LIKE' = None
     ) -> List[int]:
         return [self._measure(axis, random_state.parse_random_state(seed)) for axis in axes]
+
+    def __hash__(self) -> int:
```

Comment:
consider making this a cached_method as well.

Reply:
done

Reply:
done